boards/xtensa/esp32/Makefile.board

```diff
@@ -1,4 +1,3 @@
-# Default to xt-run but let the user override to xtsc-run if they desire
-# TODO
 EMU_PLATFORM ?=
 DEBUG_SCRIPT :=
+FLASH_SCRIPT := esp32.sh
```

Comment:
I'm planning to add a `debug` command to this script as well, to invoke OpenOCD with the right settings.